enclave/src/contracts/balance.rs

```diff
@@ -0,0 +1,213 @@
+use std::collections::{BTreeMap};
+use serde::{de::{self,Visitor}, Serialize, Deserialize, Serializer, Deserializer};
+use core::str::FromStr;
+use core::cmp::Ord;
+use std::prelude::v1::*;
+use std::{fmt,vec::Vec};
+use std::str;
```

Comment:
Please add `crate::` to all the `std` because this `std` is injected `tstd` in a `no_std` mode. If possible, better to remove `use std::prelude::v1::*` as well.